docs/user-guide/migrate-to-greptimedb/migrate-from-prometheus.md

```diff
@@ -29,3 +29,4 @@ To add GreptimeDB as a Prometheus data source in Grafana, please refer to the [G
 
 </DocTemplate>
 
+If you need a more detailed migration plan or example scripts, please provide the specific table structure and data volume. The [GreptimeDB official community](https://github.com/orgs/GreptimeTeam/discussions) will offer further support. Welcome to join the [Greptime Slack](http://greptime.com/slack).
```

Comment:
please move this line to `docs/db-cloud-shared/migrate/_migrate-from-prometheus.md`

Reply:
Why?

Reply:
This document is from the template `docs/db-cloud-shared/migrate/_migrate-from-prometheus.md`.
Only content within the template can be displayed.